intgtest/intgSpec.js

```diff
@@ -36,7 +36,7 @@ if(!SERVER || !AUTHTOKEN) {
 // Tests
 //
 describe('Vantiq SDK Integration Tests', function() {
-    this.timeout(5000);
+    this.timeout(150000000000);
```

Comment:
I assume this was for your debugging purposes, but I don't think we want to keep it this high.